gammapy/estimators/flux_point.py

```diff
@@ -870,8 +869,13 @@ def run(self, datasets, steps="all"):
             rows.append(row)
 
         table = table_from_row_data(rows=rows, meta={"SED_TYPE": "likelihood"})
-        return FluxPoints(table).to_sed_type("dnde")
 
+        flux_points = FluxPoints(table)
+        if "norm-scan" not in self.selection:
```

Comment:
Why do we need this extra case? Isn't moving the `"norm-scan"` to the optional columns sufficient?

Reply:
For some reason, the current version of the `FluxPointsEstimator` transforms the `FluxPoints` object to `dnde`.  I kept this option only for the case without the `norm-scan`. But indeed it can be removed.
Do you know why it was returning a `dnde` `FluxPoints`?

Reply:
I think for "historical" reasons, some people said it's maybe weird if the returned object does not contain any flux or dnde column. I think it's fine like this and users can just convert to any type they want with a single call to `.to_sed_type()` after...